packages/internal/src/kv-store.js

```diff
@@ -64,36 +51,25 @@ export function makeKVStore(db, ensureTxn, trace) {
 
   const sqlKVGetNextKey = db.prepare(`
     SELECT key
-    FROM kvStore
+    FROM ${tableName}
     WHERE key > ?
     LIMIT 1
   `);
   sqlKVGetNextKey.pluck(true);
 
   /**
-   * getNextKey enables callers to iterate over all keys within a
-   * given range. To build an iterator of all keys from start
-   * (inclusive) to end (exclusive), do:
+   * getNextKey enables callers to iterate over all keys within a given range.
+   * To build an iterator of all keys from start (inclusive) to end (exclusive),
+   * do:
    *
-   * function* iterate(start, end) {
-   *   if (kvStore.has(start)) {
-   *     yield start;
-   *   }
-   *   let prev = start;
-   *   while (true) {
-   *     let next = kvStore.getNextKey(prev);
-   *     if (!next || next >= end) {
-   *       break;
-   *     }
-   *     yield next;
-   *     prev = next;
-   *   }
-   * }
-   *
-   * @param {string} previousKey  The key returned will always be later than this one.
-   *
-   * @returns {string | undefined} a key string, or undefined if we reach the end of the store
+   * function* iterate(start, end) { if (kvStore.has(start)) { yield start; }
+   * let prev = start; while (true) { let next = kvStore.getNextKey(prev); if
+   * (!next || next >= end) { break; } yield next; prev = next; } }
```

Comment:
It is not acceptable to format the function like this. Will it be protected from prose-like line-wrapping if we wrap it in ````` ``` `````s?
`````suggestion
   * ```js
   * function* iterate(start, end) {
   *   if (kvStore.has(start)) {
   *     yield start;
   *   }
   *   let prev = start;
   *   while (true) {
   *     let next = kvStore.getNextKey(prev);
   *     if (!next || next >= end) {
   *       break;
   *     }
   *     yield next;
   *     prev = next;
   *   }
   * }
   * ```
`````